### ISSUE TITLE
Investigate corrupted feeds in the acceptance tests

### ISSUE THREAD

**Issue by davidgamez** [MobilityData/gtfs-validator#1848]
### Describe the bug

As part of #1837, there are 13 feeds that appeared in the acceptance test report as `corrupted`. This can happen when a validation report is not generated due to multiple causes. One of the causes is a runtime exception triggered by the validator. After conducting further investigation, I tested the feed [mdb-392](https://mobilitydatabase.org/feeds/mdb-392) against production web deployment and staging. 
- In case of production, the report was generated with few visible empty spaces in the feed information section, [report](https://gtfs-validator-results.mobilitydata.org/dc76865a-b878-4149-9e2c-bb835a4e3539/report.html). 
- In case of staging, the report was not generated and an error was visible in the logs:
```
java.lang.NullPointerException: Cannot invoke "org.mobilitydata.gtfsvalidator.table.GtfsFeedInfo.feedStartDate()" because "info" is null
```

### Steps/Code to Reproduce

Execute the GTFS validator against  the 13 feeds listed as corrupted
Corrupted marked feeds:
- [it-puglia-mobilita-e-trasporti-molfetta-gtfs-1147](https://mobilitydatabase.org/feeds/mdb-1147)
- [it-regione-autonoma-della-sardegna-collegamenti-marittimi-grimaldi-gtfs-1192](https://mobilitydatabase.org/feeds/mdb-1192)
- [it-regione-autonoma-della-sardegna-collegamenti-marittimi-moby-gtfs-1048](https://mobilitydatabase.org/feeds/mdb-1048)
- [it-regione-autonoma-della-sardegna-collegamenti-marittimi-sardinia-ferries-gtfs-1050](https://mobilitydatabase.org/feeds/mdb-1050)
- [it-regione-autonoma-della-sardegna-collegamenti-marittimi-tirrenia-gtfs-1051](https://mobilitydatabase.org/feeds/mdb-1051)
- [mexico-jalisco-direccion-general-de-transporte-publico-de-puerto-vallarta-gtfs-2034](https://mobilitydatabase.org/feeds/mdb-2034)
- [mexico-oaxaca-secretaria-de-movilidad-oaxaca-gtfs-2033](https://mobilitydatabase.org/feeds/mdb-2033)
- [mx-jalisco-secretaria-de-movilidad-del-estado-de-jalisco-gtfs-1926](https://mobilitydatabase.org/feeds/mdb-1926)
- [pe-aeroexpreso-gtfs-1985](https://mobilitydatabase.org/feeds/mdb-1985)
- [tr-kocaeli-kocaeli-gtfs-1122](https://mobilitydatabase.org/feeds/mdb-1122)
- [us-arizona-national-park-service-gtfs-1905](https://mobilitydatabase.org/feeds/mdb-1905)
- [us-ohio-athens-public-transit-gtfs-4](https://mobilitydatabase.org/feeds/mdb-4)
- [us-wisconsin-beloit-transit-gtfs-392](https://mobilitydatabase.org/feeds/mdb-392)

 

### Expected Results

All reports are generated and the extraction of the feed information should not block the generation of the HTML and JSON reports.

### Actual Results

Validation reports are not generated due to feed information logic.

### Screenshots

![Screenshot 2024-09-25 at 1 43 15 PM](https://github.com/user-attachments/assets/9311701b-6883-47b7-8d1c-bc7ab33c1945)

### Validator version

5.0.1

### Operating system

Web

**Comment by davidgamez** [comment]
Tested all `corrupted` files. 

- [it-puglia-mobilita-e-trasporti-molfetta-gtfs-1147](https://mobilitydatabase.org/feeds/mdb-1147)
  - feed_info.txt had no rows.
  - Fixed with latest commit. 
- [it-regione-autonoma-della-sardegna-collegamenti-marittimi-grimaldi-gtfs-1192](https://mobilitydatabase.org/feeds/mdb-1192)
  - feed_info.txt had no rows.
  - Fixed with latest commit. 
- [it-regione-autonoma-della-sardegna-collegamenti-marittimi-moby-gtfs-1048](https://mobilitydatabase.org/feeds/mdb-1048)
  - Feed download failed. 
- [it-regione-autonoma-della-sardegna-collegamenti-marittimi-sardinia-ferries-gtfs-1050](https://mobilitydatabase.org/feeds/mdb-1050)
  - feed_info.txt had no rows.
  - Fixed with latest commit. 
- [it-regione-autonoma-della-sardegna-collegamenti-marittimi-tirrenia-gtfs-1051](https://mobilitydatabase.org/feeds/mdb-1051)
  - Feed download failed. 
- [mexico-jalisco-direccion-general-de-transporte-publico-de-puerto-vallarta-gtfs-2034](https://mobilitydatabase.org/feeds/mdb-2034)
  - feed_info.txt had no rows.
  - Fixed with latest commit. 
- [mexico-oaxaca-secretaria-de-movilidad-oaxaca-gtfs-2033](https://mobilitydatabase.org/feeds/mdb-2033)
  - feed_info.txt had no rows.
  - Fixed with latest commit. 
- [mx-jalisco-secretaria-de-movilidad-del-estado-de-jalisco-gtfs-1926](https://mobilitydatabase.org/feeds/mdb-1926)
  - feed_info.txt had no rows.
  - Fixed with latest commit. 
- [pe-aeroexpreso-gtfs-1985](https://mobilitydatabase.org/feeds/mdb-1985)
  - feed_info.txt had no rows.
  - Fixed with latest commit. 
- [tr-kocaeli-kocaeli-gtfs-1122](https://mobilitydatabase.org/feeds/mdb-1122)
  - feed_info.txt had no rows.
  - Fixed with latest commit. 
- [us-arizona-national-park-service-gtfs-1905](https://mobilitydatabase.org/feeds/mdb-1905)
  - feed_info.txt had no rows.
  - Fixed with latest commit. 
- [us-ohio-athens-public-transit-gtfs-4](https://mobilitydatabase.org/feeds/mdb-4)
  - feed_info.txt had no rows.
  - Fixed with latest commit. 
- [us-wisconsin-beloit-transit-gtfs-392](https://mobilitydatabase.org/feeds/mdb-392)
  - feed_info.txt had no rows.
  - Fixed with latest commit.

**Comment by davidgamez** [comment]
I'm closing this issue as it was fixed with #1837.
